CMakeLists.txt

```diff
@@ -170,8 +170,7 @@ configure_file(libnaboConfig.cmake.in
 # 2- installation build #
 
 # Change the library location for an install location
-get_filename_component(NABO_LIB_NAME ${libnabo_library} NAME)
-set(libnabo_library ${CMAKE_INSTALL_PREFIX}/lib/${NABO_LIB_NAME})
+set(libnabo_library ${LIB_NAME})
```

Comment:
@davidlandry93, what was the reason for this change. This breaks builds if libnabo is compiled as a static library and installed into a catkin workspace.